ldp/alg/optimizer/opt.py

```diff
@@ -60,3 +62,14 @@ def aggregate(
     async def update(self) -> None:
         for optimizer in self.optimizers:
             await optimizer.update()
+
+
+class OptimizerClearBufferCallback(Callback):
```

Comment:
Can you rename this to `ClearOptimizerBuffersCallback`? The intent is clearer

Reply:
Also can you move this to `ldp.alg.callbacks`?

Reply:
Both are now done, thank you!